lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -83,6 +110,16 @@ def self.example_code
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :linux_sdk_install_dir,
+                                       env_name: "FL_ANDROID_LINUX_SDK_INSTALL_DIR",
+                                       description: "Install directory for Android SDK on Linux",
+                                       optional: true,
+                                       default_value: "~/android-sdk"),
```

Comment:
I'm not sure, but i think hidden folder makes more sense: `~/.android-sdk`

Reply:
On Linux you'd probably install in /usr/local/android-sdk but this isn't always writeable by the current user... Happy to change to `.android-sdk`

Reply:
done